src/applications/edu-benefits/10215/containers/ConfirmationPage.jsx

```diff
@@ -1,69 +1,115 @@
+    </h2>
+    <va-process-list uswds>
+      <va-process-list-item header="Check your eligibility">
+        <div itemProp="name">
+          <h4 className="vads-u-font-size--h5">Download and save your form</h4>
         </div>
-        <h2 className="vads-u-font-size--h3">
-          Your application has been submitted
-        </h2>
-        <p>We may contact you for more information or documents.</p>
-        <p className="screen-only">Please print this page for your records.</p>
-        <div className="inset">
-          <h3 className="vads-u-margin-top--0 vads-u-font-size--h4">
-            22-10215 Statement of Assurance of Compliance with 95 Percent
-            Enrollment Ratios Claim{' '}
-            <span className="vads-u-font-weight--normal">(Form {formId})</span>
-          </h3>
-          {fullName ? (
-            <span>
-              for {fullName.first} {fullName.middle} {fullName.last}
-              {fullName.suffix ? `, ${fullName.suffix}` : null}
-            </span>
-          ) : null}
-
-          {isValid(submitDate) ? (
-            <p>
-              <strong>Date submitted</strong>
-              <br />
-              <span>{format(submitDate, 'MMMM d, yyyy')}</span>
-            </p>
-          ) : null}
-          <va-button
-            onClick={window.print}
-            text="Print this for your records"
-            uswds
-          />
+        <div itemProp="itemListElement">
+          <p>
+            We usually process claims within 30 days. We’ll let you know by mail
+            if we need more information.
+          </p>
+          <p>
+            <va-link
+              href="https://iam.education.va.gov/"
+              text="Download VA Form 22-10216"
+              download
+            />
+          </p>
         </div>
-      </div>
-    );
-  }
-}
+      </va-process-list-item>
+      <va-process-list-item header="Check your eligibility">
+        <div>
+          <h4 className="vads-u-font-size--h5">
+            Upload the form to the VA education portal
+          </h4>
+        </div>
+        <div itemProp="itemListElement">
+          <p>
+            Visit the&nbsp;
+            <a href="/education/about-gi-bill-benefits/how-to-use-benefits/">
+              VA Education File Upload Portal (opens in a new tab)
+            </a>
+            , and upload your saved VA Form 22-10216.
+          </p>
+        </div>
+      </va-process-list-item>
+      <va-process-list-item header="Check your eligibility">
+        <div itemProp="name">
+          <h4 className="vads-u-font-size--h5">Submit your form</h4>
+        </div>
+        <div itemProp="itemListElement">
+          <p>Once uploaded, click submit to finalize your request.</p>
+        </div>
+      </va-process-list-item>
+    </va-process-list>
+    <p>
+      <va-button
+        secondary
+        text="Print this page"
+        onClick={() => window.print()}
+        data-testid="ezr-print-button"
```

Comment:
Can remove

Reply:
Can you please specify which part you want me to remove. Do you want me to remove all 4 lines are just the last line.

Reply:
Done